### PR TITLE
 gitpod: Gitpod is not non-free!

### PR DESCRIPTION
**Pull request by Kreyren**
MAINTAINERS: Squash using github

Moved gitpod from non-free to main list and updated description ^-^

Signed-off-by: Jacob Hrbek <kreyren@member.fsf.org>

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [X] Your submissions are formatted according to the following requirements:
        
``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Depends on proprietary services:

``- [Name](http://homepage/) `⚠` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Non-free software in `non-free.md`:

``- [Name](http://homepage/) `⊘ Proprietary` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``

- [X] Your additions are ordered alphabetically.
- [X] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md) and marked `⊘ Proprietary`.
- [?] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.
  - The control control panel on https://gitpod.io/settings and workspace panel https://gitpod.io/workspaces are for security reasons kept private.
  - Gitpod layers are also for security reasons kept private - Gitpod is deployed using docker and these layers are prepended and appended to every dockerfile which can be added by user to make sure that the gitpod user and required tools are available. I've made a contribution that should allow it being open-sourced, but it's still being worked on by upstream.
- [?] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [X] Your additions have their `License` main server-side `Language`/platform/requirement listed.
- [X] Any licenses you have added are in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [X] If the main link of your addition points to the program's source code, there is no need to add a duplicate _Source Code_ link. The _Demo_ and _Clients_ links are not mandatory.
  - Assuming mandatory since the source code is a separate project and TypeFox (Theia's upstream) are providing theia as gitpod.
- [X] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [X] Any software project you are adding to the list is actively maintained.

### CODE REVIEW

**Comment by Kreyren** [comment]
To elaborate based on my understanding:

[Theia](https://github.com/eclipse-theia/theia) (the vscode-like editor) is created by [TypeFox](https://www.typefox.io/) where Gitpod (also maintained by TypeFox) is stable release of theia.

It should be probably mensioned somewhere on gitpod.io to avoid confusion though O.o

**Comment by Kreyren** [comment]
Note that the demo (https://gitpod.io/#https://github.com/awesome-selfhosted/awesome-selfhosted/pull/2104) requires a GitHub/Gitlab account.

Not sure if that should be mensioned somewhere~

**Comment by Kreyren** [comment]
From CI:

```
Issues :-(

> Links 

  All OK ✓

> Dupes 

  1. [L2] https://github.com/eclipse-theia/theia

The command "if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then if [ -f temp.md ]; then awesome_bot temp.md --allow-redirect --skip-save-results --allow 202 --white-list airsonic.github.io/docs/apps; else (exit 0); fi else awesome_bot *.md --allow-redirect --skip-save-results --allow 202 --white-list airsonic.github.io/docs/apps; fi" exited with 1.
```

No idea what that means~

File: README.md [hunk 1]
```diff
@@ -1360,6 +1360,7 @@ See **[Ticketing](#ticketing)**
 - [Babelfish](https://github.com/bblfsh/bblfshd) - Self-hosted server for source code parsing. It can parse any file, in any supported language, extract an Abstract Syntax Tree from it, and convert it to a Universal Abstract Syntax Tree which can enable further analysis and transformation. `GPL-3.0` `Go`
 - [Code-Server](https://coder.com/) - Visual Studio Code in the browser, hosted on a remote server. ([Source Code](https://github.com/cdr/code-server)) `MIT` `Nodejs/Docker`
 - [Eclipse Che](http://www.eclipse.org/che/) - Open source workspace server and cloud IDE. ([Source Code](https://github.com/eclipse/che)) `EPL-1.0` `Docker/Java`
+- [Gitpod](https://gitpod.io/) - Open source vscode-like and vscode extension compatible cloud IDE provided as upstream [theia](https://github.com/eclipse-theia/theia) distribution with automated self-hosting deployment for your infrastructure. ([Demo](https://gitpod.io/#https://github.com/awesome-selfhosted/awesome-selfhosted), [Source Code](https://github.com/gitpod-io/self-hosted)) `EPL-2.0` `Typescript/Go/Docker`
```
Comment (nodiscc) [inline README.md:1363]: This description makes no sense... :) Please reuse the description from https://www.gitpod.io/ (_Online IDE for GitHub and GitLab_). The inline link to theia was what was causing CI failures

Comment (nodiscc) [reply README.md:1363]: Sorry for the delay

Comment (Kreyren) [reply README.md]: Resolved in [https://github.com/awesome-selfhosted/awesome-selfhosted/pull/2104/commits/98aa8320b3dd577c16cab17ddee419f422033697 ](https://github.com/awesome-selfhosted/awesome-selfhosted/pull/2104/commits/98aa8320b3dd577c16cab17ddee419f422033697)

**Pull request merged**